docs/deployment/deployment-modes.md

```diff
@@ -3,43 +3,36 @@ title: Deployment modes
 
-## Single-node
+## Cluster mode (multi-node)
 
-This is the simplest way to get started with Quickwit. First, launch all the services with the `quickwit run` [command](../reference/cli.md), and then you're ready to ingest and search data.
-
-## Multi nodes
-
-You can deploy Quickwit on multiple nodes. We provide a [helm-chart](kubernetes.md) to help you deploy Quickwit on Kubernetes.
-
-As soon as you are running at least 2 nodes, there are several restrictions:
-- you have to use a distributed data store such as Amazon S3 or MinIO for storing your index; a local file system storage will not work;
-- if you use the [Ingest API](../reference/rest-api.md), you must send your queries directly to the indexer. When sent to a searcher, you will get a 404 response. Note that when search queries are addressed to an indexer, it acts as a root searcher node and dispatches leaf requests to searchers;
+You can deploy Quickwit on multiple nodes. We provide a [helm-chart](kubernetes.md) to help you deploy Quickwit on Kubernetes. In cluster mode, you must store your index data on a shared storage backend such as Amazon S3 or MinIO.
 
 ## One indexer, multiple searchers
 
-One Quickwit node running on a decent instance can ingest data at speeds up to 40 MB/sec from Kafka. A deployment with one indexer is thus a good start. However, you may need several searchers for handling large datasets or serving many resource-intensive queries such as aggregation queries.
+One indexer running on a small instance (4 vCPUs) can ingest documents as fast as 25 MB/s (2+ TB/day). A deployment with one indexer is thus a good place to start. However, you may need several searchers to handle large datasets or serve many resource-intensive requests such as aggregation queries.
 
 ## Multiple indexers, multiple searchers
 
-Indexing a single [data source](../configuration/source-config.md) on several indexers is currently only possible with a [Kafka source](../configuration/source-config.md#kafka-source).
-Support distributed indexing for Pulsar and the Ingest API sources is planned for Q2, stay tuned!
+Indexing a single [data source](../configuration/source-config.md) on several indexers is only possible with a [Kafka source](../configuration/source-config.md#kafka-source).
+Support distributed indexing for Pulsar and the Ingest API is planned for Quickwit 0.7 (Q3). Stay tuned!
 
 ## File-backed metastore limitations
 
-The file-backed metastore is mainly useful for testing purposes. Though it may be convenient for some specific use cases, we strongly encourage you to use a PostgreSQL metastore in production.
+The file-backed metastore is mainly designed and intended for testing purposes. Though it may be convenient for some specific use cases, we strongly recommend using a PostgreSQL metastore in production.
```

Comment:
I think we should review that sentence. For small use cases, it should be a viable way of deploying quickwit.
A file-backed metastore can work well typically in a standalone mode. If we can ensure that we have only one writer, I think it's also fine when you don't have much ingest. You end up with mestastore service that writes update sent by one indexer + janitor.